scandium-core/src/main/java/org/eclipse/californium/scandium/dtls/EcdhPskServerKeyExchange.java

```diff
@@ -110,19 +111,20 @@
 	 * @throws NullPointerException if any of arguments ecdhe, 
 	 * clientRandom, serverRandom are {@code null}
 	 */
-	public EcdhPskServerKeyExchange(String hint, ECDHECryptography ecdhe, Random clientRandom, Random serverRandom, 
+	public EcdhPskServerKeyExchange(String pskHint, ECDHECryptography ecdhe, Random clientRandom, Random serverRandom, 
```

Comment:
when you rename the parameter, don't forget to adapt the JavaDoc accordingly ...
In this context: most of the parameters lack a proper description i the JavaDoc comment and it is unclear, when to expect a `GeneralSecurityException` to be thrown. For a user, all of this is vital information in order to be able to use this class properly, don't you think?

Reply:
done